tests/console/system_prepare.pm

```diff
@@ -44,12 +44,18 @@ sub run {
     }
 
     # Register the modules after media migration, so it can do regession
-    if (get_var('SCC_ADDONS') && get_var('MEDIA_UPGRADE') && get_var('KEEP_REGISTERED')) {
+    if (get_var('SCC_ADDONS') && get_var('MEDIA_UPGRADE') && get_var('REREG_AF_UPG')) {
```

Comment:
REREG_AF_UPG seems not good in my view, not block issue.

Reply:
Any suggestion?

> REREG_AF_UPG seems not good in my view, not block issue.

Any suggestion?